database/layer.go

```diff
@@ -25,13 +25,13 @@ import (
 )
 
 const (
-	FieldLayerIsValue           = "layer"
+	fieldLayerIsValue           = "layer"
 	FieldLayerID                = "id"
 	FieldLayerParent            = "parent"
 	FieldLayerSuccessors        = "successors"
 	FieldLayerOS                = "os"
-	FieldLayerInstalledPackages = "adds"
-	FieldLayerRemovedPackages   = "removes"
+	fieldLayerInstalledPackages = "adds"
+	fieldLayerRemovedPackages   = "removes"
 	FieldLayerEngineVersion     = "engineVersion"
```

Comment:
Break these into two different blocks of consts: one exported, one not. Then put a comment above the ones that are not exported and explain why.